src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -657,6 +657,18 @@ public virtual int? SomeDacField{ get; set; }</value>
   <data name="PX1076Title" xml:space="preserve">
     <value>This code calls Acumatica internal API marked with PXInternalUseOnlyAttribute which is not intended for public use</value>
   </data>
+  <data name="PX1077Fix" xml:space="preserve">
+    <value>Change the accessibility modifier to {0}</value>
+  </data>
+  <data name="PX1077Fix_RemoveExplicitInterface" xml:space="preserve">
+    <value>Remove the explicit interface signature</value>
+  </data>
+  <data name="PX1077Title" xml:space="preserve">
+    <value>Event handlers should be protected and virtual</value>
```

Comment:
The format argument should be used here too.
```cs
Event handlers should be {0}
```
where {0} can be:
- public or public virtual
- protected or protected virtual

Reply:
Done.